save-backend/src/main/kotlin/com/saveourtool/save/backend/repository/LnkOrganizationTestSuiteRepository.kt

```diff
@@ -0,0 +1,61 @@
+package com.saveourtool.save.backend.repository
+
+import com.saveourtool.save.entities.*
+import com.saveourtool.save.permission.Rights
+import org.springframework.data.jpa.repository.Modifying
+import org.springframework.data.jpa.repository.Query
+import org.springframework.data.repository.query.Param
+import org.springframework.stereotype.Repository
+import org.springframework.transaction.annotation.Transactional
+
+/**
+ * Repository of [LnkOrganizationTestSuite]
+ */
+@Repository
+interface LnkOrganizationTestSuiteRepository : BaseEntityRepository<LnkOrganizationTestSuite> {
+    /**
+     * @param organization
+     * @return [LnkOrganizationTestSuite] by [organization]
+     */
+    fun findByOrganization(organization: Organization): List<LnkOrganizationTestSuite>
+
+    /**
+     * @param organization
+     * @param testSuite
+     * @return [LnkUserProject] by [organization] and [testSuite]
```

Comment:
@return **[LnkOrganizationTestSuite]** by [organization] and [testSuite]

Reply:
Oh sure

Reply:
Fixed comment